### PR TITLE
Kie BPMS Plugin 

### PR DESCRIPTION
**Pull request by odaghagheleh**
Kie BPMS Plugin  was updated to support new widget (progress bar)

### CODE REVIEW

**Comment by entando-service** [comment]
Can one of the admins verify this patch?

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmCaseProgressWidgetAction.java [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    private CaseManager caseManager;
+    private IKieFormManager formManager;
+    private IKieFormOverrideManager kieFormOverrideManager;
+    private IBpmWidgetInfoManager bpmWidgetInfoManager;
+    String processPath;
+    String processPathDefaultValue;
+    String casesDefinitions;
+    List<String> process;
+
+    String _frontEndMilestonesData;
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmCaseProgressWidgetAction.java]: Remove underscores in variable names. Its all over the codebase but trying to normalize to java standards for new stuff or changed code.

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmCaseProgressWidgetAction.java [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+            }
+            this.setProcess(cids);
+            this.setProcessPathDefaultValue(processPath);
+        } catch (ApsSystemException ex) {
+            Logger.getLogger(BpmCaseProgressWidgetAction.class.getName()).log(Level.SEVERE, null, ex);
+        }
+        return SUCCESS;
+    }
+
+//    public String chooseCaseForm() throws ApsSystemException {
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmCaseProgressWidgetAction.java]: Can this be removed or is it needed for future dev.

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/specialWidget.xml [hunk 3]
```diff
@@ -122,4 +129,47 @@
...
+
+        <action name="changeForm*" class="jpkiebpmBpmCaseProgressWidgetAction" method="changeForm">
+            <result type="tiles">admin.jpkiebpm.page.widget.bpmCaseProgress.config</result>
+            <result name="input" type="tiles">admin.jpkiebpm.page.widget.bpmCaseProgress.config</result>
+            <interceptor-ref name="entandoDefaultStack">
+                <param name="requestAuth.requiredPermission">managePages</param>
+            </interceptor-ref>
+        </action>        
+        
+<!--        <action name="chooseCaseForm*" class="jpkiebpmBpmCaseProgressWidgetAction" method="chooseCaseForm">
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/specialWidget.xml]: Remove?

File: plugins/entando-plugin-jpkiebpm/src/main/resources/spring/plugins/jpkiebpm/aps/frontActionsDefs.xml [hunk 4]
```diff
@@ -19,6 +19,12 @@
...
     </bean>
+    
+     <bean id="jpkieBpmCaseProgressFrontAction"
+          class="org.entando.entando.plugins.jpkiebpm.aps.internalservlet.BpmCaseProgressAction"
+          parent="abstractBaseAction" scope="prototype" >
+        <property name="caseManager" ref="jpkiebpmsCaseManager"/>   
+    </bean>
 	
 	
 <!--
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/resources/spring/plugins/jpkiebpm/aps/frontActionsDefs.xml]: remove?

**Pull request merged**